service/dap/server_test.go

```diff
@@ -3970,7 +3973,7 @@ Type help followed by a command for full documentation.
 					checkEval(t, got, formatConfig(50, false, false, "", false, [][2]string{}), noChildren)
 
 					// Read and modify showGlobalVariables.
-					client.EvaluateRequest("dlv config showGlobalVariables", 1000, "repl")
+					client.EvaluateRequest("dlv config -list showGlobalVariables", 1000, "repl")
```

Comment:
This is an unfortunate change. Adding -list is much less intuitive and more typing for the user.  But adding special empty filter  to separate look-up and reset seems complicated too :(

Reply:
Yeah I agree that its not ideal :( Though it lines up better with the dlv cli now.